chapters/object_types.txt

```diff
@@ -53,6 +53,15 @@ Feature support information can be queried as properties on the `ANARIRenderer`
 |===================================================================================================
 
 
+Feature: `KHR_RENDERER_VOLUME_METHOD`
```

Comment:
Maybe this could be multiple extensions one per method all of which interact with this parameter? That way the availability of the method can be queried as a feature while introducing the ability to select it in case of multiple options.

`KHR_RENDERER_VOLUME_METHOD_RAYCAST`, `KHR_RENDERER_VOLUME_METHOD_SPLAT` etc.
All of which define new values for the method parameter but none specify a default. In case the parameter is not set the choice is up to the implementation.